pkg/providers.bzl

```diff
@@ -34,12 +34,12 @@ PackageFilesInfo = provider(
     fields = {
         "attributes": """Attribute information, represented as a `dict`.
 
-        Keys are strings representing attribute identifiers, values are
-        arbitrary data structures that represent the associated data.  These are
-        most often strings, but are not explicity defined.
+Keys are strings representing attribute identifiers, values are
```

Comment:
Did you dedent this on purpose?

Reply:
Yes.  I spent an annoying amount of time misreading this a few weeks back, so I made it more obvious.

I have no problem reverting this, but the structure of the provider `fields` can look like a large string blob if you aren't reading it closely.  Once upon a time, a colleague of my tried something like this:

```python
    dict(
        attributes = """Attribute information, represented as a `dict`.

        Keys are strings representing attribute identifiers, values are
        arbitrary data structures that represent the associated data.  These are
        most often strings, but are not explicity defined.

        For known attributes and data type expectations, see the Common
        Attributes documentation in the `rules_pkg` reference.
        """,
        ...
    )
```

Github's default syntax highlighting doesn't make this easy to distinguish, but with color styles where identifiers are vastly different this is much easier to read.

WDYT?  Nothing we need to worry about in the here and now, though.